lib/wallaby/query.ex

```diff
@@ -29,6 +29,7 @@ defmodule Wallaby.Query do
     * `:count` - The number of elements that should be found (default: 1).
     * `:visible` - Determines if the query should return only visible elements (default: true).
     * `:text` - Text that should be found inside the element (default: nil).
+    * `:get` - The number of the element to return if multimple elements satisfy the selection criteria.
```

Comment:
Not a big fan of lazy typists over here, code is written once but read many times so i favor the expressiveness :) Let's wait a bit of time to see if keath gets time to check in here and add his voice :)

For me `get` reminds me a bit too much of HTTP GET requests which confuses me in thsi context.